registrations/permissions.py

```diff
@@ -170,3 +172,31 @@ def get_signups_keys():
             return "presence_status" not in data_keys
 
         return False
+
+
+class CanAccessPriceGroups(RegistrationBasePermission):
+    def has_permission(self, request: Request, view: APIView) -> bool:
+        if not request.user.is_authenticated:
+            return False
+
+        if request.user.is_superuser:
+            return True
+
+        if request.method == "POST":
+            (
+                __,
+                user_organization,
+            ) = self.user_data_source_and_organization_from_request(request)
+            return request.user.is_financial_admin_of(user_organization)
+
+        return True
+
+    def has_object_permission(
+        self, request: Request, view: APIView, obj: PriceGroup
+    ) -> bool:
+        if view.action == "retrieve":
```

Comment:
When checking whether the request is a read or a write operation in custom permissions, [the preferred way in DRF is checking against `permissions.SAFE_METHODS`](https://www.django-rest-framework.org/api-guide/permissions/#custom-permissions) .

Reply:
Changed as suggested